lib/flame/pool.ex

```diff
@@ -713,25 +731,31 @@ defmodule FLAME.Pool do
     }
   end
 
-  defp async_boot_runner(%Pool{on_grow_start: on_grow_start, name: name} = state) do
-    {strategy_module, strategy_opts} = state.strategy
+  # Starts runners asynchronously as to not block the pool. Note that boot_max_concurrency
+  # does not bound the number of concurrent booting runners here.
+  defp async_boot_runner(%Pool{on_grow_start: on_grow_start, name: name} = state, opts \\ []) do
+    runner_opts = Keyword.get(opts, :runner_opts)
 
+    # :count should be the _total_ count you want to spawn, not the delta
+    new_count = Keyword.get_lazy(opts, :count, fn -> desired_count(state) end)
```

Comment:
When would `:count` already be defined in the opts? Guessing this is for extensibility so we can pass it in later if needed?

Reply:
Its included when we call async_boot_runner from boot_min_runners to finish booting the remaining runners